server/sonar-web/src/main/js/components/SourceViewer/SourceViewerBase.tsx

```diff
@@ -240,35 +207,40 @@ export default class SourceViewerBase extends React.PureComponent {
 
   fetchComponent() {
     this.setState({ loading: true });
-    const loadIssues = (component, sources) => {
-      this.props.loadIssues(this.props.component, 1, LINES, this.props.branch).then(issues => {
-        if (this.mounted) {
-          const finalSources = sources.slice(0, LINES);
-          this.setState(
-            {
-              component,
-              issues,
-              issuesByLine: issuesByLine(issues),
-              issueLocationsByLine: locationsByLine(issues),
-              loading: false,
-              notAccessible: false,
-              notExist: false,
-              hasSourcesAfter: sources.length > LINES,
-              sources: this.computeCoverageStatus(finalSources),
-              sourceRemoved: false,
-              symbolsByLine: symbolsByLine(sources.slice(0, LINES))
-            },
-            () => {
-              if (this.props.onLoaded) {
-                this.props.onLoaded(component, finalSources, issues);
+    const loadIssues = (component: SourceViewerFile, sources: SourceLine[]) => {
+      this.props.loadIssues!(this.props.component, 1, LINES, this.props.branch).then(
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Forbidden non null assertion ([typescript:S2966](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS2966))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fstas%2Fpopups&issues=AWHb6qYww6XGHKvPuQTD&open=AWHb6qYww6XGHKvPuQTD)

[ISSUE_KEY]: AWHb6qYww6XGHKvPuQTD